sdk/cosmosdb/cosmos/test/internal/unit/timeoutFailoverRetryPolicy.spec.ts

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
```

Comment:
Thanks for adding these detailed unit tests, Please add functional tests (maybe using plugins to mock service failures) also.

Reply:
updated